dashboards/TDinsight.json

```diff
@@ -5604,5 +5570,5 @@
   "timezone": "",
   "title": "TDinsight",
   "uid": "tdinsight",
-  "version": 32
+  "version": 19
```

Comment:
why the version become smaller one?

Reply:
The version is not seriously meaningful to grafana, it will always be 1 when imported at user side.

I use it equal to revision version as https://grafana.com/grafana/dashboards/15167/revisions .